ssetests/tests_basic_parsing.go

```diff
@@ -1,19 +1,67 @@
 package ssetests
 
 import (
+	"crypto/rand"
```

Comment:
Can probably use `math/rand` if you don't need this to be cryptographically secure. I'll just assert that it'll be more performant with no evidence.